spec/app/domain/authentication/constraints/non_permitted_constraint_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe(Authentication::Constraints::NonPermittedConstraint) do
+  let(:non_permitted_constraint) { ["iat", "nbf", "exp", "iss"] }
```

Comment:
@tzheleznyak could you use the MANDATORY_CLAIMS_DENY_LIST const? 
it has more claims and single-source-of-truth

Reply:
Its a test so i don't think its really a problem. But already @nessiLahav  adding this to host so when he will add i will use it. Even in another pr i will do the usage. So there won't be two variables